assignments/assignment-12/if-else-math-random.js

```diff
@@ -0,0 +1,29 @@
+/* ----- 1 ----- */
+
+var randomNumber = Math.floor(Math.random() * 10) + 1;
+var userName = "Alexandria";
+console.log(randomNumber);
+
+if (randomNumber >= 1 && randomNumber <= 3){
+    console.log(userName.slice(0, 3));
```

Comment:
You should not console.log, but assigned some name to `name` variable. And that name should be used in message. Now, your message will have only "Abbos" name.